actors/verifreg/src/state.rs

```diff
@@ -23,13 +28,42 @@ impl State {
             .flush()
             .map_err(|e| anyhow::anyhow!("Failed to create empty map: {}", e))?;
 
+        let empty_mapmap = MapMap::new(store, HAMT_BIT_WIDTH, HAMT_BIT_WIDTH).root().map_err(|e| anyhow::anyhow!("Failed to create empty multi map: {}", e))?;
+
         Ok(State {
             root_key,
             verifiers: empty_map,
             verified_clients: empty_map,
             remove_data_cap_proposal_ids: empty_map,
+            allocations: empty_mapmap,
+            next_allocation_id: 0,
+            claims: empty_mapmap,
         })
     }
 }
+#[derive(Serialize_tuple, Deserialize_tuple, Clone, Debug, PartialEq)]
+pub struct Claim {
+    pub provider: Address,
+    pub client: Address,
+    pub data: Cid,
+    pub size: u64,
+    // The min period which the provider must commit to storing data
+    pub term_min: ChainEpoch,
+    // The max period for which provider can earn QA-power for the data
+    pub term_max: ChainEpoch,
+    pub term_start: ChainEpoch,
+    pub sector: SectorID, 
+}
+
+#[derive(Serialize_tuple, Deserialize_tuple, Clone, Debug)]
+pub struct Allocation {
+    pub client: Address,
+    pub provider: Address,
+    pub data: Cid,
+    pub size: u64,
```

Comment:
I *think* this (and above) needs to be PaddedPieceSize. I think we need to constrain to the sector's power-of-two chunk in order to later do inclusion proofs. This will match the DealProposal it comes from.